include/TrustWalletCore/TWEthereumChainID.h

```diff
@@ -21,6 +21,7 @@ enum TWEthereumChainID {
     TWEthereumChainIDVeChain = 74,
     TWEthereumChainIDThunderToken = 108,
     TWEthereumChainIDTomoChain = 88,
+    TWEthereumChainIDSmartChain = 97,
```

Comment:
I miss the Binance reference here, maybe TWEthereumChainIDBinanceSmartChain or TWEthereumChainIDBinanceSC

Reply:
Ok